### PR TITLE
Last 3 tasks

### PR DESCRIPTION
**Pull request by toti0git**
please check the code.

### CODE REVIEW

File: Java/OtherTasks345.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            case 11:
+                System.out.println("November");
+                break;
+            case 12:
+                System.out.println("December");
+                break;
+            default:
+                System.out.println("Please, enter the number between 1 and 12");
+        }
+
```
Comment (gavrillina) [inline Java/OtherTasks345.java:95]: delete empty lines please

File: Java/OtherTasks345.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        System.out.println("Enter the length of array");
+        Scanner sc = new Scanner(System.in);
+        int amount = sc.nextInt();
+        int sum = 0;
+        int[] arr = new int[amount];
+
+        System.out.println("Enter the elements of array");
+        for (int i = 0; i <= arr.length - 1; i++) {
+            arr[i] = sc.nextInt();
+            sum += arr[i];
```
Comment (gavrillina) [inline Java/OtherTasks345.java:44]: add please multiplication for task

File: Java/OtherTasks345.java (hunk 1 above)
Comment (toti0git) [reply Java/OtherTasks345.java:95]: ok, fixed

File: Java/OtherTasks345.java (hunk 2 above)
Comment (toti0git) [reply Java/OtherTasks345.java:44]: fixed

**Pull request closed without merging**